libs/ldmk-transport/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@ledgerhq/ldmk-transport",
```

Comment:
shouldn't it be named `@ledgerhq/live-dmdk` ?

Reply:
indeed the package should be called `@ledgerhq/live-dmk`